doc/sphinx/language/core/basic.rst

```diff
@@ -134,8 +134,9 @@ Keywords
   used as identifiers::
 
     _ Axiom CoFixpoint Definition Fixpoint Hypothesis Parameter Prop
-    SProp Set Theorem Type Variable as at cofix discriminated else end
-    fix for forall fun if in let match return then where with
+    SProp Set Theorem Type Variable as at cofix else
+    end fix for forall fun if in let match
+    return then where with
```

Comment:
If you maintain `end` on the same line as `else` and `return` .. on the previous line, we will see that all this PR does (compared to the previous one) is to remove `discriminated` from the list.

Reply:
True.  It was different before the rebase of this PR in the last day.  I didn't realize you were removing keywords from the doc in #12172.  Was that intentional and perhaps in anticipation of certain PRs being merged soon?  Prior to 12172, the list was:

```
    _ Axiom CoFixpoint Definition Fixpoint Hypothesis IF Parameter Prop
    SProp Set Theorem Type Variable as at by cofix discriminated else
    end exists exists2 fix for forall fun if in lazymatch let match
    multimatch return then using where with
```

From which `IF by exists exists2 lazymatch multimatch` and `using` have been removed as well as the statement "(omitting 25 specialized tokens that begin with ``#int63_``)" which AFAICT are still defined as tokens in master.

Or look just above this comment: https://github.com/coq/coq/pull/12172#discussion_r418490166

Reply:
>Was that intentional and perhaps in anticipation of certain PRs being merged soon?

This was done late in answer to a comment by Hugo. Based on the (maybe mistaken) impression that this was supposed to be the list of keywords defined outside the plugin. But as I said, I don't think we should differentiate the keywords defined in plugins from keywords defined in notations. We should differentiate only between in prelude / not in prelude.

>as well as the statement "(omitting 25 specialized tokens that begin with #int63_)" which AFAICT are still defined as tokens in master

Indeed, this was done in anticipation of #12122. If this PR doesn't get merged, we should revert that.

Reply:
> If you maintain end on the same line as else and return .. on the previous line, we will see that all this PR does (compared to the previous one) is to remove discriminated from the list.

Done.